### DISCUSSION TITLE
How can we support the project?

### DISCUSSION THREAD

**Discussion by LeonardoAcevedoC** [GitFiend/Support#201] [General]
Hi, i was wondering if there is a way to donate or something... ive been using gitfiend for a while on Linux Mint it helped me a lot because i wasnt able to use github desktop on linux, it just wouldnt let me... i found gitfiend and has it has been working without a problem, is actually easy to use and works fine.

**Comment by GitFiend** [comment]
Hey thanks, thanks nice to hear. There isn't anything right now, but I'm thinking about making the source available and setting up something.

**Comment by dhruvhaldar** [comment]
I'm interested in donating too. I've been using it for a while, and it's been a breeze.
